app/models/vm_scan/dispatcher.rb

```diff
@@ -246,20 +246,6 @@ def get_eligible_proxies_for_job(job)
         return []
       end
 
-      if @vm.requires_storage_for_scan?
-        if @vm.storage.nil?
-          msg = "Vm [#{@vm.path}] is not located on a storage, aborting job [#{job.guid}]."
-          queue_signal(job, {:args => [:abort, msg, "error"]})
-          return []
-        else
-          unless @vm.storage.supports?(:smartstate_analysis)
-            msg = @vm.storage.unsupported_reason(:smartstate_analysis)
-            queue_signal(job, {:args => [:abort, msg, "error"]})
-            return []
-          end
-        end
-      end
-
```

Comment:
So https://github.com/ManageIQ/manageiq-providers-ovirt/pull/653/files#diff-24aea1f97754a89593e8311e8ea2bf2d897a159336d1f1c578d17fbb1964506dR10 (I think rightly) moves the storage type check into the VM's `supports?(:smartstate_analysis)`, we still need to check `@vm.supports?(:smartstate_analysis)` somewhere?  Is this done elsewhere?